resip/recon/UserAgent.cxx

```diff
@@ -569,6 +610,79 @@ UserAgent::destroySubscriptionImpl(SubscriptionHandle handle)
    }
 }
 
+void 
+UserAgent::createPublicationImpl(PublicationHandle handle, const Data& status, const Data& eventType, const NameAddr& target, unsigned int publicationTime, const Mime& mimeType)
+{
+   // Ensure we have a client publication handler for this event type
+   if(!mDum.getClientPublicationHandler(eventType))
+   {
+      mDum.addClientPublicationHandler(eventType, this);
+   }
+   // Ensure that the request Mime type is supported in the dum profile
+   if(!mProfile->isMimeTypeSupported(PUBLISH, mimeType))
+   {
+      mProfile->addSupportedMimeType(PUBLISH, mimeType);  
```

Comment:
OK, in that case, you can leave it like that

@sgodin should a PUBLISH client need to call ```addSupportedMimeType``` given that is is only sending messages?